overview/default.nix

```
@@ -412,6 +414,16 @@ let
       <body>
         ${args.content}
         <script>
+          // On document load, put all elements into the DOM that can be used with JS only
+          document.addEventListener("DOMContentLoaded", () => {
+            document.querySelectorAll("template").forEach(template => {
+                if (template.hasAttribute("scripted")) {
```

```suggestion
            document.querySelectorAll("template[scripted]").forEach(template => {
```

I think could can simplify this by folding the conditional into the query